Dockerfile.compile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:jammy
+RUN apt-get update && apt-get install -y \
+    openjdk-11-jdk \
```

Comment:
Wondering if shouldn't run on 17 vs. 11 kinda thing for builds now.  The target on the gradle plugin should still compile to 11... but can use the 17 JDK for this.

Reply:
NOTE:  This is how other projects work, so separate PR for that.